src/client/templates/card.html

```diff
@@ -4,7 +4,7 @@
             {{#with cardInfo this.id}}
             <span class="white-text">{{text}}</span>
             <div class="card-action">
-                <div class="chip" id="deleteCardButton">
+                <div class="chip" id="deleteCardButton" data-toggle="modal" data-target="#{{_id}}DeleteModal">
```

Comment:
I'll be honest, I have no idea how this is working, not saying it's bad, obviously it works, I just have no idea lol. Please explain next time we do code review